app/assets/javascripts/channels/attendance.js

```diff
@@ -0,0 +1,8 @@
+(function() {
+  App.chat = App.cable.subscriptions.create("AttendanceChannel", {
+    connected: function() {},
+    disconnected: function() {},
+    received: function(data) {alert("Student with ID" + data.student_id + "Marked"+ data.attendance)}
```

Comment:
I'm not sure how we will do this, but I don't think we want it to be an alert because that can be irritating if the teacher is trying to do a lot of these at a time.

Reply:
This was just for testing / demonstration that it works -- to provide a place to start from